app/forms/candidate_interface/degree_completion_status_form.rb

```diff
@@ -12,6 +12,12 @@ def save(degree)
       degree.update!(predicted_grade: grade_is_predicted?)
     end
 
+    def save_and_update(degree)
```

Comment:
This is so nitty that i genuinely don't care, but we have an established pattern of using `update` when we have two persistence based methods in forms.

Reply:
absolutely no worries, have just fixed 👍